pkg/utils/release/sbom.go

```diff
@@ -0,0 +1,86 @@
+/* SBOM of type structures matches Pyxis structure
+
+When SBOM files are uploaded to Pyxis it's required to be modified according to
+Pyxis's requirment where the CamelBack method does not match,
```

Comment:
Another typo: -> `requirements`

But this whole thing actually doesn't read great. I would propose:
```
When SBOM components are uploaded to Pyxis, key names have to be modified
to conform to GraphQL naming conventions. 
1. Use _ instead of camel case, e.g. camelCase -> camel_case
2. Use _ instead of -, e.g. key-with-dash -> key_with_dash
See https://github.com/redhat-appstudio/release-service-utils/blob/main/pyxis/upload_sbom.py
```

Reply:
awesome, 
your's does reads much better